### ISSUE TITLE
fates_hdim_levleaf should be a real

### ISSUE THREAD

**Issue by rgknox** [NGEET/fates#872]
It looks like this history variable: [fates_hdim_levleaf](https://github.com/NGEET/fates/blob/sci.1.57.0_api.23.0.0/main/FatesInterfaceTypesMod.F90#L244) should be a real, no?

The values that it indicates are the cumulative LAI+SAI, which is a real, not an integer.

**Comment by ckoven** [comment]
Yes, good catch.  I think that must be left over from when dinc was hard-coded to be 1.
